tests/foreman/cli/test_contentview.py

```diff
@@ -1310,17 +1297,17 @@ def test_positive_add_puppet_module(self):
         )
         # Check output of `content-view info` subcommand
         content_view = ContentView.info({'id': content_view['id']})
-        self.assertGreater(len(content_view['puppet-modules']), 0)
-        self.assertEqual(puppet_module['name'], content_view['puppet-modules'][0]['name'])
+        assert len(content_view['puppet-modules']) > 0
```

Comment:
Using `assert len() > 0 ` is redundant, because `len` will return 0 (false) or greater than zero (true)

Reply:
@mshriver so this should just be removed?

Reply:
Its optional and up to you, but worth mentioning. 

The types in python have a truthy-ness to them. If `content_view['puppet-modules']` is a list, you don't even have to call `len()`, you can assert directly against the list.  If the list is empty the assertion will fail, as an empty list is 'falsy'. If there are any elements in the list at all, the assertion will pass.  Going that far makes an assumption that `content_view['puppet-modules']` is a list, but the same assertion logic applies if its a dictionary, or a string - an empty dict or string will fail the direct assert, a dictionary with anything in it, or a string of any length (even whitespace) will pass the assertion.

You may find it more valuable to have a fully explicit assertion here against the length, but personally I would write my assertions to be direct here.

```In [6]: assert [], 'list was empty'                                             
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-6-d507d26ab449> in <module>
----> 1 assert [], 'list was empty'

AssertionError: list was empty
```

```
In [9]: assert [False], 'list was empty' 
```